content/en/examples/admin/konnectivity/egress-selector-configuration.yaml

```diff
@@ -18,4 +18,4 @@ egressSelections:
       # The other supported transport is "tcp". You will need to set up TLS 
       # config to secure the TCP transport.
       uds:
-        udsName: /etc/srv/kubernetes/konnectivity-server/konnectivity-server.socket
+        udsName: /etc/kubernetes/konnectivity-server/konnectivity-server.socket
```

Comment:
Can you keep original `/etc/srv/kubernetes/konnectivity-server/` directory to be consistent with other [configurations](https://github.com/kubernetes/kubernetes/search?q=%2Fetc%2Fsrv%2Fkubernetes%2Fkonnectivity-server) in k8s?

Reply:
But this will make configuration inconsistent with the other kubeadm services, which store all configuration in [`/etc/kubernetes`](https://github.com/kubernetes/kubernetes/search?q=%22%2Fetc%2Fkubernetes%22&type=) and for which the original instructions were written.

Reply:
That looks ok. Sorry for the delay.